servers/display_server_headless.h

```diff
@@ -168,11 +168,10 @@ class DisplayServerHeadless : public DisplayServer {
 	Error file_dialog_with_options_show(const String &p_title, const String &p_current_directory, const String &p_root, const String &p_filename, bool p_show_hidden, FileDialogMode p_mode, const Vector<String> &p_filters, const TypedArray<Dictionary> &p_options, const Callable &p_callback) override { return ERR_UNAVAILABLE; }
 
 	void release_rendering_thread() override {}
-	void make_rendering_thread() override {}
```

Comment:
I don't know where this override came from, it isn't in the main display server, and hasn't been for some time AFAIK